### DISCUSSION TITLE
五笔输入法 86 极点版，配置方案分享

### DISCUSSION THREAD

**Discussion by KyleBing** [rime/home#766] [Show and tell]
五笔用户是否一直在转为 macOS 系统后找不到合适的输入法而烦恼？
相信这个能解决你的大部分烦恼。

地址： [https://github.com/KyleBing/rime-wubi86-jidian](https://github.com/KyleBing/rime-wubi86-jidian)

- 五笔86极点码表
- 简入繁出
- 大写数字
- 造词工具（for macOS）

## 皮肤

macOS
<img width="500px" src="https://user-images.githubusercontent.com/12215982/101584309-56833e80-3a18-11eb-9b83-a43204ad363c.png">

Windows
<img width="400" alt="auto-create-words" src="https://user-images.githubusercontent.com/12215982/127078734-41584c0f-96b8-4630-8cd4-3522d9ce384e.jpg">

<img width="400" alt="auto-create-words" src="https://user-images.githubusercontent.com/12215982/127078737-9b6fb1c9-3b32-42e9-970f-baeb68c024b0.jpg">

<img width="400" alt="auto-create-words" src="https://user-images.githubusercontent.com/12215982/127078741-c4eb7b20-2199-4eea-a6da-491ff620a695.jpg">

## 输入
<img width="800px" src="https://user-images.githubusercontent.com/12215982/81524061-6f444c80-9382-11ea-9ecc-edb38a2cfc3f.gif">

**Comment by oishi81** [comment]
我就是使用的@kyleBing大佬的作品，很棒！

**Comment by chenrylee** [comment]
请问候选界面的圆角，在Windows下可以实现么？配置文件中我没有看到相关的设置呢。

**Reply by KyleBing** [reply]
我现在都不知道 windows 皮肤要怎么设置，现在工作用 windows 皮肤还是乱的，也一直没处理。不太清楚。

**Reply by LEOYoon-Tsaw** [reply]
不能，需改程式

**Reply by oishi81** [reply]
corner_radius 这个参数设置了无效，不知道什么原因

**Reply by chenrylee** [reply]
这里是否可以请教 @lotem 呢。。。
